test/parse.test.js

```diff
@@ -173,7 +173,7 @@ describe(`parse`, function () {
       p.process(
         `const thing = "face";
   <template>Hi`,
-        "path/to/my/component.gjs",
```

Comment:
is the old way still able to be supported?

Reply:
Upthread I said a breaking change is OK, so I don't think we should keep the old way. It's a very easy upgrade.

Reply:
excellent!